webapp/locales/de.json

```diff
@@ -316,66 +316,67 @@
         "reports": {
             "empty": "Glückwunsch, es gibt nichts zu moderieren.",
             "name": "Meldungen",
-            "status": "Aktueller Status",
-            "content": "Inhalt",
-            "author": "Autor",
-            "decision": "Entscheidung",
-            "enabled": "Entsperrt",
-            "disabled": "Gesperrt",
-            "decided": "Entschieden",
-            "noDecision": "Keine Entscheidung!",
-            "decideButton": "Bestätige",
-            "DecisionSuccess": "Erfolgreich entschieden!",
-            "enabledBy": "Entsperrt von",
-            "disabledBy": "Gesperrt von",
-            "previousDecision": "Vorherige Entscheidung:",
-            "enabledAt": "Entsperrt am",
-            "disabledAt": "Gesperrt am",
+            "reporter": "gemeldet von",
+            "submitter": "gemeldet von",
+            "disabledBy": "deaktiviert von",
             "reasonCategory": "Kategorie",
             "reasonDescription": "Beschreibung",
-            "submitter": "Gemeldet von",
-            "numberOfUsers": "{count} Nutzern",
+            "status": "",
```

Comment:
I see a lot of code reverted in this PR. Have the translations not been updated on Lokalise?

Reply:
We have to import both, English and German in Lokalise to avoid this. Thank you for being aware